pkg/virt-launcher/virtwrap/cli/libvirt.go

```diff
@@ -243,9 +247,12 @@ func (l *LibvirtConnection) installWatchdog(checkInterval time.Duration) {
 				return
 
 			case <-time.After(checkInterval):
-				l.reconnectIfNecessary()
-
-				alive, err := l.Connect.IsAlive()
+				var alive bool
+				var err error
+				err = l.reconnectIfNecessary()
```

Comment:
oops... yeah we should have been checking this for an error all along. good catch